app/js/views/service.jsx

```diff
@@ -33,13 +34,16 @@ export default class Service extends BaseView {
       return null;
     }
 
-    switch(this.state.service.type) {
+    switch (this.state.service.type) {
       case 'ip-camera@link.mozilla.org':
         return (<CameraServiceView service={this.state.service}
-                                   foxbox={this.foxbox} />);
+                                   foxbox={this.foxbox}/>);
+      case 'philips_hue@link.mozilla.org':
+        return (<LightServiceView service={this.state.service}
+                                  foxbox={this.foxbox}/>);
       default:
         return (<DefaultServiceView service={this.state.service}
-                                    foxbox={this.foxbox} />);
+                                    foxbox={this.foxbox}/>);
```

Comment:
__question:__ why do you remove space before ```/>```? I was usually adding them to separate it from the value of the last attribute (haven't checked w3c spec on this though, but browsers are not restrictive at all). Let's agree on the same approach! What do you prefer? :)